test/chai.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * In the browser, importing chai/chai.js will register to the window. In webpack it will
+ * be parsed as a commonjs module, so it won't be on the window. In this file we conditionally
+ * export from the window, or a webpack specific module import.
+ */
```

Comment:
So is this a workaround for web-test-runner or just something we did so wtr and karma could work side-by-side? Or something else?

Reply:
Yep, web-test-runner expects everything to be packaged as ESM. The `chai` package doesn't have such an entry point, that's why we need this wrapper file.